velox/docs/functions/spark/array.rst

```diff
@@ -196,6 +196,25 @@ Array Functions
         SELECT size(NULL, true); -- -1
         SELECT size(NULL, false); -- NULL
 
+.. spark:function:: slice(array(E), start, length) -> array(E)
+
+    | Returns a subarray starting at 1-based index ``start`` or from end if negative, with ``length`` elements.
+    | Return empty array if ``start`` point outside of the array.
+    | Return elements between ``start`` and the end of the array if ``start + length`` is outside of the array.
+    | Return empty array if ``length`` is 0.
+    | Throws exception if ``start`` is 0.
+    | Throws expcetion if ``length`` is negative.
+    
+    ::
+
+            SELECT slice(array(1, 2, 3, 4), 2, 2); -- [2, 3]
```

Comment:
nit: too many spaces. Perhaps follow the document format of below examples.

https://github.com/facebookincubator/velox/blob/0358c1c87f2fef7205e60a400b35bf62708a703d/velox/docs/functions/spark/array.rst?plain=1#L218-L224

Reply:
If using line breaker `|`, I'd have to add indent here to ensure code block works

Now I use the second option(directly append an empty line for each sentence), so removed indent here